### PR TITLE
merge to resolve conflict

### PR DESCRIPTION
**Pull request by FriGo74**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -2,8 +2,12 @@
 <html>
 <head>
     <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>{% block title %} TN-Client {% endblock %}</title>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```
Comment (BlackTurtle123) [inline templates/base.html]: Is this second time loading in the css needed?
Local copy not enough?

File: templates/home.html [hunk 2]
```diff
@@ -7,40 +7,107 @@
...
-        <p>
-            <button id="send_tn" type="button">Send
-                $TN
-            </button>
-        </p>
-    </form>
-    <h3>Transactions History</h3>
-    <button data-addr={{ address }} id="tx_load">Show tx history</button>
+   
+  <html>
```
Comment (BlackTurtle123) [inline templates/home.html]: HTML and BODY tage included in the base already, do we really need it here?

**Review by BlackTurtle123 — CHANGES_REQUESTED** [review]
.DS_Store included:
Add to gitignore
Delete them

Double bootstrap included:
Is online version needed?

Double html tags, is it needed to add a body and html tag when he gets these already from the base.html?

File: templates/base.html (hunk 1 above)
Comment (FriGo74) [reply templates/base.html]: If local is bootstrap  4.3.1. it can go.

File: templates/home.html (hunk 2 above)
Comment (FriGo74) [reply templates/home.html]: But base.html closes the body and html tag as well not sure whats correct we can remove opening and closing tags from home.html see what happens.

**Review by BlackTurtle123 — APPROVED** [review]

**Pull request merged**